### DISCUSSION TITLE
How to customize plugin selection drawer

### DISCUSSION THREAD

**Discussion by WeitingC219** [react-page/react-page#1166] [Q&A]
I'm wondering if it is possible to replace the default plugin selection drawer and plugin adding button with custom components. 
Any examples?

<img width="1604" alt="Screen Shot 2022-02-25 at 3 22 17 PM" src="https://user-images.githubusercontent.com/52922454/155673034-e27c1730-3806-45bb-9f51-829e97397935.png">

**Comment by macrozone** [comment]
currently not, but its planned to make more components replaceable. What is the use case?

**Comment by WeitingC219** [comment]
Thanks, just for style consistency while integrating react-page with the existing system.

**Reply by ledahu05** [reply]
We have the same need. 
To be more precise, we might have hundred of plugins and we will need to improve UX and the search to deal with the number of plugins.
